custom/mgh_epic/tasks.py

```diff
@@ -0,0 +1,15 @@
+from django.conf import settings
+
+from celery.schedules import crontab
+
+from corehq.apps.celery import periodic_task
+
+from custom.mgh_epic.sync_epic_appointments import sync_all_appointments_domain
+
+MGH_EPIC_DOMAINS = settings.CUSTOM_DOMAINS_BY_MODULE['custom.mgh_epic']
+
+
+@periodic_task(run_every=crontab(hour="*", minute=1), queue="background_queue")
+def sync_all_epic_appointments():
+    for domain in MGH_EPIC_DOMAINS:
+        sync_all_appointments_domain(domain)
```

Comment:
This should have error handling to avoid exceptions in one domain from affecting others

Reply:
fixed here: https://github.com/dimagi/commcare-hq/pull/35125/commits/6b772baa6ad5daacbc7667324519ac78c8a8c2bf